activestate.macos.yaml

```diff
@@ -0,0 +1,70 @@
+constants:
+  - name: SYSTRAY_APP
+    if: eq .OS.Name "MacOS"
+    value: ActiveState\ Desktop.app
+scripts:
+  - name: build-systray-app
+    language: bash
+    standalone: true
+    if: eq .OS.Name "MacOS"
+    description: Copies the MacOS application directory to the build dir
+    value: |
+      mkdir -p build/${constants.SYSTRAY_APP}
+
+      cp -r installers/${constants.SYSTRAY_APP} build/
+  - name: generate-update
```

Comment:
This seems almost identical to the non-macos version. Can we instead use a constant to tweak the flags that are different per OS?